packages/storybook-react/src/stories/template/PageStep-2.stories.tsx

```diff
@@ -20,101 +25,241 @@ import {
-            <FieldsetLegend>NIEUWE ADRES</FieldsetLegend>
-            <div role="radiogroup" aria-labelledby="nieuwe-adres-label" data-rating-value="">
-              <Heading4 id="nieuwe-adres-label">Hoe wilt u zoeken *</Heading4>
-              <FormField>
-                <RadioButton id="postcode" name="woonplaats-en-straat" />
-                <FormLabel htmlFor="postcode">Postcode</FormLabel>
-              </FormField>
-              <FormField>
-                <RadioButton id="woonplaats-en-straat" name="woonplaats-en-straat" />
-                <FormLabel htmlFor="woonplaats-en-straat">Woonplaats en Straat</FormLabel>
-              </FormField>
+    const onNewAddressSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+      event.preventDefault();
+
+      if (dateRef.current && dateRef.current.value === '') {
+        setInvalid(true);
+        dateRef.current.setAttribute('aria-invalid', 'true');
+      }
+      if (postcodeRef.current && postcodeRef.current.value === '') {
+        console.log(postcodeRef.current.value, 'test');
+        setInvalid(true);
+        postcodeRef.current.setAttribute('aria-invalid', 'true');
+      }
+      if (homeNumberRef.current && homeNumberRef.current.value === '') {
+        setInvalid(true);
+        homeNumberRef.current.setAttribute('aria-invalid', 'true');
+      }
+    };
+
+    const validateField = (current: HTMLInputElement | null) => {
+      if (current && current.value) {
+        if (current.getAttribute('aria-invalid') && current.validity.valid) {
+          current.removeAttribute('aria-invalid');
+          setInvalid(false);
+        } else if (current && current.value === '' && !current.validity.valid) {
+          current.setAttribute('aria-invalid', 'true');
+          setInvalid(true);
+        }
+      }
+    };
+
+    const onNewAddressChange = () => {
+      validateField(dateRef.current);
+      validateField(postcodeRef.current);
+      validateField(homeNumberRef.current);
+    };
+
+    return (
+      <Page {...args} style={pageStyle}>
+        <PageHeader>
+          <Link href="/" aria-label="Gemeente Utrecht">
+            <UtrechtLogo />
+          </Link>
+        </PageHeader>
+        <PageContent role="main">
+          <BreadcrumbNav appearance="arrows" style={{ marginBlockEnd: '1.5rem' }}>
+            <BreadcrumbLink href="https://example/" index={0} current={false} rel="home">
+              Home
+            </BreadcrumbLink>
+            <BreadcrumbLink href="https://example/" index={0} current={true} rel="Online loket">
+              Online loket
+            </BreadcrumbLink>
+          </BreadcrumbNav>
+          <HeadingGroup>
+            <Heading1>Een verhuizing doorgeven</Heading1>
+            <Paragraph>Stap 2 van 5 — De verhuizing</Paragraph>
+          </HeadingGroup>
+          <div style={rowStyle}>
+            <div style={descriptionStyle}>
+              <Heading2>De verhuizing</Heading2>
+              <Separator />
```

Comment:
Separator should only be used for semantic grouping, not visual effect. To improve the situation you could use `<Separator role="presentation"/>`